src/components/characterList.jsx

```diff
@@ -10,17 +11,7 @@ const CharacterList = ({ currentScript }) => {
       return;
     }
     // fetch request to get the character data
-    try {
-      const response = await fetch('/api/script/' + currentScript + '/characters');
-
-      if (response.ok) {
-        setCharacters(await response.json());
-      } else {
-        console.error(`server error: ${response.body} when fetching character data`);
-      }
-    } catch (error) {
-      console.error(`network error: ${error} when fetching character data`);
-    }
+      setCharacters( await getCharacters(currentScript))
```

Comment:
nit: run code formatter to fix extra space

Reply:
Also, this code isn't equivalent to what was here before and now has a bug:  
Previously, if the fetch failed for one reason or another, then `setCharacters()` wouldn't be called.  
Now, `setCharacters()` is always called and passed the result of `await getCharacters()`. This passes invalid data when the fetch fails.